processor/src/main/java/com/instagram/common/json/annotation/processor/JsonParserClassData.java

```diff
@@ -43,6 +43,7 @@
   private final boolean mPostprocessingEnabled;
   private final String mValueExtractFormatter;
   private final String mParentInjectedClassName;
+  private final boolean mUseGetters;
 
   public JsonParserClassData(
```

Comment:
at this point, this class is a total mess.  someone who comes along to touch this code is at high risk of breaking something.  a ton of this information is coming straight from TypeData.  can this be refactored to accept an instance of TypeData?